### PR TITLE
docs: MaaS introduction enhancement

### PR DESCRIPTION
**Pull request by antongisli**
## Describe the Change

This PR aims to make a more succinct and understandable introduction to the pcg 

## Review Changes

💻 [Preview](https://deploy-preview-1272--docs-spectrocloud.netlify.app/clusters/data-center/maas/install-manage-maas-pcg)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1272#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (ritawatson) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:22]:
```suggestion
- The PCG then facilitates all communication between Palette and MAAS, enabling Palette to create new clusters on machines that MAAS provides.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (ritawatson) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:17]: I would move lines 24 and 26 here as an intro to your process bullet list below:

"Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box. You can use the installer on any Linux x86-64 system with a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 

You can set up the PCG as a single- or three-node cluster based on your requirements for high availability (HA)."

**Review by ritawatson — APPROVED** [review]
This is a good addition to the doc - thanks Anton. I suggest moving a couple lines ahead of the process list being added. I also suggested a few wording and styling changes.

LGTM with suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1272#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1272#pullrequestreview-1409278749" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ritawatson** [comment]
That's fine Anton. It's your call.

**Comment by cloudmaniac** [comment]
FYI, I have a couple of notes from when I tried the MAAS PCG deployment last week. I'll add my commits tomorrow or Friday.

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (cloudmaniac) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:19]:
```suggestion
- Run the PCG installer on a laptop, workstation, or jump box.
```

**Comment by karl-cardenas-coding** [comment]
@cloudmaniac we should remove the steps in the UI and instead link to the docs page. No one is really maintaining the UI text so it could lead to outdated instructions.

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:128]:
```suggestion
5. The installer installs the MAAS machines and uses the configuration file to build a new cluster to host the PCG application.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:387]:
```suggestion
Two new nodes will be added to the PCG cluster.

<br />

```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:237]:
```suggestion
The **/opt/spectrocloud** folder is volume mapped to the installer's **/tmp** folder.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:227]:
```suggestion
  <br />
  
  <WarningBox>

  Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and 
  Resource Pool combination.

  </WarningBox>
```

**Pull request merged**
